README.md

```diff
@@ -1,24 +1,61 @@
-# README
+# Hello-react-rails
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> Description of the project.
 
-Things you may want to cover:
+In this exercise I had to connect previous knowledge with new knowledge as I get a chance to use Webpack with `jsbundling-rails gem`.
 
-* Ruby version
+## Built With
 
-* System dependencies
+* Ruby on Rails
+* Webpack
+* PostgreSQL
+* React
 
-* Configuration
+## Live Demo
 
-* Database creation
+(Not available)
 
-* Database initialization
+### Prerequisites
 
-* How to run the test suite
+Tu run this project in your local environment, You will need to have the following tools installed loally :
 
-* Services (job queues, cache servers, search engines, etc.)
+* `Ruby 3.0+`
+* `Rails 6.0+`
+* `PostgreSQL` version 14+
 
-* Deployment instructions
+## Set up
 
-* ...
+* `git clone git@github.com:christianbiring1/hello-rails-react.git`
+* `cd Budget_App`
+* `bundle install`
+* `rails db:create`
+* `rails db:migrate`
+* `rails db:seed`
+* `rails s`
+* Open up the `localhost` in your browser
+
```

Comment:
Thank you for this tip @cwaku, It really helps me a lot. Because at first, I had a difficult time refreshing my react component on the views. `npm run build` is doing the whole thing now :smile: 
Thank you so much @cwaku

Reply:
@christianbiring1 You are welcome. I'm glad it was helpful